apps/client/src/app/modules/list/list-details-panel/list-details-panel.component.ts

```diff
@@ -91,13 +99,24 @@ export class ListDetailsPanelComponent implements OnChanges, OnInit {
 
   hasAlreadyBeenOpened: boolean;
 
+  private server$: BehaviorSubject<string> = new BehaviorSubject<string>(undefined);
+  serializationHelper = new ListRowSerializationHelper(this.i18nTools, this.l12n, this.gt);
+
   constructor(private i18nTools: I18nToolsService, private l12n: LocalizedDataService,
-              private message: NzMessageService, private translate: TranslateService,
-              private dialog: NzModalService, private listsFacade: ListsFacade,
-              private itemPicker: ItemPickerService, private listManager: ListManagerService,
-              private progress: ProgressPopupService, private layoutOrderService: LayoutOrderService,
-              private eorzeaFacade: EorzeaFacade, private alarmsFacade: AlarmsFacade,
-              public settings: SettingsService, private lazyData: LazyDataService) {
+    private message: NzMessageService, private translate: TranslateService,
+    private dialog: NzModalService, private listsFacade: ListsFacade,
+    private itemPicker: ItemPickerService, private listManager: ListManagerService,
+    private progress: ProgressPopupService, private layoutOrderService: LayoutOrderService,
+    private eorzeaFacade: EorzeaFacade, private alarmsFacade: AlarmsFacade,
+    public settings: SettingsService, private lazyData: LazyDataService,
+    private authFacade: AuthFacade,
+    private gt: GarlandToolsService,
+    private _clipboardService: ClipboardService
+  ) {
+    //hold onto server name once resolved
+    this.authFacade.mainCharacter$.pipe(
+      map(char => char.Server)
+    ).subscribe(this.server$);
```

Comment:
Instead of doing this, you can just have `server$` be an `Observable` and make it 

```ts
    this.authFacade.mainCharacter$.pipe(
      map(char => char.Server)
    )
```

The general rule is to avoid subscription, because unhandled subscriptions = memory leaks.